components/utilities/dialog/index.jsx

```diff
@@ -372,13 +389,16 @@ const Dialog = createReactClass({
 	},
 
 	render () {
-		let style = {};
+		const style = {};
```

Comment:
See master branch. This needs to `let`. `Object.assign` was removed and spread operator was put in it's place.